src/styles/globals.css

```diff
@@ -6,4 +6,15 @@ body,
 #__next,
```

Comment:
Let's migrate this .css to .scss. I KNOW! we have removed .scss file just last PR :D. But it was in favor of tailwind. 

IF we will need to use something else than tailwind (like this case) we must use scss modules (just nextjs docs) not plain old terrible global CSS :D